df.history.xml

```diff
@@ -824,94 +824,112 @@
-        <enum-item name='bring_thralldom_to_the_world'/>
-        <enum-item name='bring_torture_to_the_world'/>
-        <enum-item name='provide_opportunities_for_acts_of_valor'/>
-
-        <enum-item name='bring_war_to_the_world'/>
-        <enum-item name='find_relative'/>
-        <enum-item name='offer_condolences'/>
-        <enum-item name='be_brought_to_safety'/>
-        <enum-item name='help_with_rescue'/>
-        <enum-item name='insufficient_work'/>
-        <enum-item name='work_request'/>
-        <enum-item name='make_weapon'/>
-        <enum-item name='vent_at_boss'/>
-        <enum-item name='cry_on_boss'/>
-
-        <enum-item name='should_have_reached_goal'/>
-        <enum-item name='insufficient_progress_toward_goal'/>
-        <enum-item name='going_wrong_direction'/>
-        <enum-item name='arrived_at_location'/>
-        <enum-item name='entity_no_longer_rules'/>
-        <enum-item name='left_site'/>
-        <enum-item name='reunited_with_loved_one'/>
-        <enum-item name='violent_disagreement'/>
-        <enum-item name='adopted'/>
-        <enum-item name='true_name_invocation'/>
-
-        <enum-item name='arrived_at_person'/>
-        <enum-item name='eradicate_beasts'/>
-        <enum-item name='entertain_people'/>
-        <enum-item name='make_a_living_as_a_warrior'/>
-        <enum-item name='study'/>
-        <enum-item name='flight'/>
-        <enum-item name='scholarship'/>
-        <enum-item name='be_with_master'/>
-        <enum-item name='become_citizen'/>
-        <enum-item name='prefers_working_alone'/>
-
-        <enum-item name='jealousy'/>
-        <enum-item name='glorify_hf'/>
-        <enum-item name='have_not_performed'/>
-        <enum-item name='prevented_from_leaving'/>
-        <enum-item name='curiosity'/>
-        <enum-item name='hire_on_as_mercenary'/>
-        <enum-item name='hire_on_as_performer'/>
-        <enum-item name='hire_on_as_scholar'/>
-        <enum-item name='drink'/>
-        <enum-item name='admire_architecture'/>
-
-        <enum-item name='pray'/>
-        <enum-item name='relax'/>
-        <enum-item name='danger'/>
-        <enum-item name='cannot_find_artifact'/>
-        <enum-item name='failed_mood'/>
-        <enum-item name='lack_of_sleep'/>
-        <enum-item name='trapped_in_cage'/>
-        <enum-item name='great_deal_of_stress'/>
-        <enum-item name='unable_to_leave_location'/>
-        <enum-item name='sanctify_hf'/>
-
-        <enum-item name='artifact_is_heirloom_of_family_hfid'/>
-        <enum-item name='cement_bonds_of_friendship'/>
-        <enum-item name='as_a_symbol_of_everlasting_peace'/>
-        <enum-item name='on_a_pilgrimage'/>
-        <enum-item name='gather_information'/>
-        <enum-item name='seek_sanctuary'/>
-        <enum-item name='part_of_trade_negotiation'/>
-        <enum-item name='artifact_is_symbol_of_entity_position'/>
+        <enum-item name='insurrection' caption='for an insurrection'/>
```

Comment:
I'm pretty sure `caption` isn't a recognized attribute, so this won't show up anywhere.
If you want code to be able to access these, a better option would be to use enum-attrs, like this: https://github.com/DFHack/df-structures/blob/8d3191569199b8d65304e688046ea7766c6b3621/df.job-types.xml#L24-L45